### PR TITLE
Updated aws cli in silta-cicd images

### PR DESCRIPTION
**Pull request by agnis-mateuss**
Updated aws cli to latest available

### CODE REVIEW

File: silta-cicd/circleci-php7.2-node14-composer1/Dockerfile [hunk 1]
```diff
@@ -28,10 +28,16 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.
   && sudo apt-get clean
 
 # Install AWS cli and aws-iam-authenticator
-RUN sudo apt install -y awscli git python3 \
-  && curl -o /tmp/aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.18.9/2020-11-02/bin/linux/amd64/aws-iam-authenticator \
-  && chmod +x /tmp/aws-iam-authenticator \ 
-  && sudo mv /tmp/aws-iam-authenticator /bin/aws-iam-authenticator
+RUN sudo apt install -y git python3
```
Comment (Rade333) [inline silta-cicd/circleci-php7.2-node14-composer1/Dockerfile:31]: It's unnecessary to have the installation of git and python3 in a separate RUN line, please move them as part of the same RUN line as below, which is also installing unzip and curl.

Having it separate creates an extra image layer.

**Review by Rade333 — CHANGES_REQUESTED** [review]

**Comment by Rade333** [comment]
@agnis-mateuss when you are back, please see the requested changes for this PR :)

File: silta-cicd/circleci-php7.2-node14-composer1/Dockerfile (hunk 1 above)
Comment (agnis-mateuss) [reply silta-cicd/circleci-php7.2-node14-composer1/Dockerfile]: Grouped up package installs and increased image version

**Review by Rade333 — CHANGES_REQUESTED** [review]
Looking good otherwise but there's still one more cicd image that is the newest: circleci-php8.2-node20-composer2

**Review by Rade333 — APPROVED** [review]

**Pull request merged**